src/utils/formHelperFunctions.ts

```diff
@@ -145,7 +145,11 @@ export const getAllElements = (step: any) => {
   ];
 };
 
-export const lookUpTrigger = (step: any, elementID: any, elementType: any) => {
+export const lookUpTrigger = (
+  step: any,
+  elementID: any,
+  elementType: any
+): Record<string, any> => {
```

Comment:
The return object for this function has the 'elementType' prop instead of the 'type' prop specified in the 'Trigger' type defined in Form.ts.  This return is used as the trigger that is part of ContextOnAction.  This mismatch seems to be an old issue though because it is also an issue with **OnError and onSubmit**.

Reply:
changed elementType to type here